src/bin/vip-import-sql.js

```diff
@@ -48,7 +48,7 @@ command( {
 	envContext: true,
 	module: 'import-sql',
 	requiredArgs: 1,
-	requireConfirm: 'Are you sure you want to import the contents of the provided SQL file?',
+	requireConfirm: 'Are you sure you want to proceed?',
```

Comment:
I think the message we had before is more descriptive than this one ("proceed" vs "import the content of the provided SQL file")

Reply:
Yeah, I changed the wording because I added more confirmation prompts to the logging, so this initial one seemed a bit misleading (we're not importing the file right away- we do some checks first, and then we have a final prompt asking if you want to continue with the import).

Not super important, but I thought leaving it simpler was better. I changed it back to the original message for now- we can always play around with the wording!